Ullaakut/Bloggo
# Grid layout (#72)

## Goal of this PR

Fixes #72 

Creating a basic grid layout in order to get started on the front-end.
This PR creates a raw responsive grid layout which falls back to the mobile version if the grids are not supported by the browser.


## Testing

1. Open and compile `grid.scss`.
2. Open `index.html` in Google Chrome, Mozilla Firefox, Microsoft Edge, Opera, Apple Safari.
3. In each of the aforementioned browsers check responsiveness by changing the width of the window (above and below 768px).
4. Check if, for the browsers which aren't supporting the grid layout, the app is displayed in a mobile version.


## Screenshots

![image](https://user-images.githubusercontent.com/38378975/45552590-bdadea80-b831-11e8-83ba-d4eb0058cb0f.png)
Mobile version

![image](https://user-images.githubusercontent.com/38378975/45552644-ecc45c00-b831-11e8-975c-254691f1330e.png)
Desktop version
